lib/FeedItem.php

```diff
@@ -428,6 +428,37 @@ public function setUid($uid) {
 		return $this;
 	}
 
+	/**
+	 * Get duration (for items where the link points to media)
+	 *
+	 * Use {@see FeedItem::setDuration()} to set the duration.
+	 *
+	 * @return int Duration in seconds.
+	 */
+	public function getDuration() {
+		return $this->duration;
```

Comment:
Also, it should be documented in https://github.com/RSS-Bridge/rss-bridge/blob/master/docs/05_Bridge_API/02_BridgeAbstract.md#item-parameters.